core/src/main/java/nu/yona/server/analysis/service/WeekActivityDTO.java

```diff
@@ -0,0 +1,39 @@
+package nu.yona.server.analysis.service;
+
+import java.time.LocalDate;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import nu.yona.server.analysis.entities.WeekActivity;
+
+@JsonRootName("weekActivity")
+public class WeekActivityDTO extends IntervalActivityDTO
```

Comment:
Added another class with that name. This class will remain because it maps one to one with `WeekActivity` (which is the activity for a specific goal) and will be grouped into a `WeekActivityOverviewDTO`.